workspaces/diff-engine/tests/domain-conflict-scenarios-generated.rs

```diff
@@ -0,0 +1,114 @@
+// DO NOT EDIT! This file was generated via workspace-scripts/diff-engine/generate-tests
+use insta::assert_debug_snapshot;
+use optic_diff_engine::{diff_interaction, HttpInteraction, SpecEvent, SpecProjection};
+use petgraph::dot::Dot;
+use serde_json::json;
+
+#[test]
+pub fn scenario_9949a8cc69a0063a70f5ae98672d8c23a1068a2b2d08f8bb7b9bbd968ec29f0d() {
```

Comment:
The intent of these being content-addressable is so we can add useful descriptions separately (that reference the content address) and know whether the input data has changed since that would potentially invalidate the snapshot or description.